cmd/powcfg/transformations.go

```diff
@@ -0,0 +1,19 @@
+package main
```

Comment:
Most of the code in this PR is structure.

Here is a transformation You receive a `*StorageConfig`, modify it if applies to whatever you want to do, and return if this entry should be re-saved (since was modified). The query, saving, etc is done by boilerplate code.

From now forward, whenever we want to make some massive change, we should only create a function here and point `applyTransformation` in `main.go` to use it.